src/components/AreaContainer/AreaContainer.js

```diff
@@ -0,0 +1,10 @@
+import  React from 'react';
+import './AreaContainer.css';
+
+const AreaContainer = () => {
```

Comment:
I'm about to take on the styling of the Area Container - we'll merge that either tonight or tomorrow depending on what I can get done.